### PR TITLE
adds homepage news: no announcement on june 9th

### PR DESCRIPTION
**Pull request by SBBCornell**
quick screenshot:
Screen Shot 2020-06-05 at 4 12 28 PM

### CODE REVIEW

**Comment by jweiskoff** [comment]
Isn't this usually done as a bulleted list?

File: browse/templates/home/news.html [hunk 1]
```diff
@@ -17,6 +17,9 @@
 16 Mar 2020: <a href="https://blogs.cornell.edu/arxiv/2020/03/16/arxiv-announces-its-first-executive-director/">arXiv announces its first executive director</a><br/>
 12 Mar 2020: <a href="https://blogs.cornell.edu/arxiv/2020/03/12/arxiv-responds-to-covid-19-uncertainties/">arXiv responds to COVID-19 uncertainties</a><br/>
 {%- endif -%}
+
+<h4 class="homepage-news-title">News</h4>
+Notice to authors and readers that there will be no regular announcement of papers on Tuesday 9 June 2020. Please see the <a href="https://arxiv.org/new/#5June2020">news post for details</a>.<br>
```
Comment (bdc34) [inline browse/templates/home/news.html:22]: Looks good. The note at /new#5June2020 is live so that is good.

**Comment by bdc34** [comment]
> Isn't this usually done as a bulleted list?

It doesn't look like it has been recently:
`08 Jan 2020: Congratulations to arXiv founder Paul Ginsparg for being awarded the <a href="http://news.cornell.edu/stories/2020/01/cis-professor-and-arxiv-founder-receives-physics-award">2020 Karl Taylor Compton Medal for Leadership in Physics<a/><br/>`

or

`06 Jan 2020: Martin Luther King, Jr. Holiday Schedule <a href="/new/index#6Jan2020">announced</a><br/>`

**Comment by jimentwood** [comment]
Could do relative links in the future... Not worth changing for now
https://arxiv.org/new/#5June2020
new/#5June2020

**Pull request merged**